content/english/episodes/ep-8.md

```diff
@@ -0,0 +1,97 @@
+publish_date: 2025-04-15T00:00:00-07:00
+podcast_file: "/audio/hidden-figures-ep8.mp3"
+podcast_duration: 1647.6
+episode_image: "images/ep8-mojdeh.png"
+podcast_bytes: "66615526" # the length of the episode in bytes
+images: ["images/ep8-mojdeh.png", "images/mojdeh.jpg"]
+explicit: false 
+type: 'episode'
+episode: '8'
+season: 1
+transcript: '/transcript/hidden-figures-of-python-ep8.srt'
+episode_type: full
+layout: episode
+youtube_id: "Gh1OOv2qtTA"
+  
+speaker:
+  url: "/speakers/mojdeh-rastgoo/"
+  title: "Mojdeh Rastgoo"
+  image: "/images/mojdeh.jpg"
+  description: "
+Mojdeh Rastgoo is a Machine Learning Data Scientist and Project Manager based in Pris.
+She has a PhD in computer sicence and has been an active memnber of the community since 2018. 
+She is a co organizer of PyLadies Paris and member of PSF Code of Conduct Working-Group since mid 2023.
+And recently she has joined the PyPodcats team as the fifth organizer.  
+
+She participated in organization of EuroSciPy 2023/2024 conference, JupyterCon 2023, PyCon.DE & PyData Berlin 2024, and PyLadies Paris meetups since 2022. 
+She has also given introductory python tutorials in several occaisions.
+"
+  social:
+  - name: github
+    icon: fa-brands fa-github
+    link: https://github.com/mrastgoo
+
+  - name: twitter
+    icon: fa-brands fa-x-twitter
+    link: https://x.com/RastgooMojdeh
+
+  - name: linkedin
+    icon: fa-brands fa-linkedin
+    link: https://www.linkedin.com/in/mojdeh-rastgoo-6462753a/
+
+  - name: web
+    icon: fa-solid fa-globe
+    link: https://mrastgoo.github.io./
+  
+---
+
+We interviewed Mojdeh Rastgoo the new memner of PyPodcats.
+
+Since discovering Python and open source communities, Mojdeh has been actively involved in the python community, since she gave her first tutorial in 2018.
+She is a member of PSF CoC working group, co organizer of PyLadies Paris and now PyPodcast host!!
+
+In this chat, Mojdeh tells us more about herself and her passion about the community.
+In addition we give a short recap of 2024, and talk about our plans for 2025. 
+We also introduce couple of new changes in our organization, such as open-collective account, wehere you can support us.
+
+Be sure to listen to the episode to hear about our plans and get to know your new host Mojdeh !
+
+## Topic discussed
+
+- Introductions
+- What are we up to nowadays 
+- Getting to know Mojdeh 
+- What Saint-Gobain does and Mojdeh roles in Saint-Gobain
+- How did she ended up in Europe 
+- Her involvement in the community
+- PyLadies Paris
+- Public speaking and Lightning talks
+- Rols of LLM in today's life 
+- Personal projects 
+- Growth mindset and changing perspectives
+- Personal interests and hobbies 
+- Podcats reflection and plans for the future
+- Founding and supports
+
+
+## Links from the show
+
+- https://pypodcats.live/blog/kickstart_2025/
+- https://opencollective.com/pypodcats
+
```

Comment:
Maybe add a few more links:
- Erasmus Mundus scholarship
- links to episodes of Vicky, Joanna, and Jessica (because they were mentioned during the session)
- Think Again book
- PyPodcats Open Collective